lib/elixir_newbie_web/live/blog_show.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ElixirNewbieWeb.BlogShow do
+  use ElixirNewbieWeb, :live_view
+  alias ElixirNewbie.Blog
+  alias ElixirNewbie.Stats
+  alias ElixirNewbie.Blog.Post
+  ## alias ElixirNewbie.Router.Helpers, as: Routes ##
+
+
+  def render(assigns) do ## BigSpaces Wonders... about embedding the  {/if}  <figure class="makeup-blog"> as in v1
+    ~H"""
+     <section class="bg-[url('/images/BlogBackground.webp')] min-h-screen bg-repeat text-white">
+      <div class="to-black/60 bg-gradient-to-t from-black">
+        <.navigation/>
+
+
+      <section class="mx-auto w-2/3 pb-24">
+        <h1 class="mb-10 block rounded-lg text-4xl hover:opacity-75"><%= raw @blog.title%></h1>
+        <p class="mb-6 text-2xl"><%= raw @blog.description %></p>
+        <p class="mb-6 text-l">By <%= raw @blog.author_name %>. Written on <%= Date.to_string(@blog.date) %> </p>
+        <img class="mb-4 rounded-2xl" src={"/images/posts/#{@blog.cover_image}"}/>
+        <%!-- {#if @blog.livebook_url}
```

Comment:
Make sure to remove all commented code. I think this is a WIP so it's not too bad to have them in, but generally, try not to commit commented code.